include/core/parameters_lagrangian.h

```diff
@@ -350,22 +365,21 @@ namespace Parameters
     {
     public:
       // Choosing grid motion type
+      // forces motion type is something that ForceTorqueOnWall set when
+      // enable is at true
       enum class MotionType
       {
         translational,
         rotational,
+        forces,
         none
       } motion_type;
 
       // Translational velocity of the moving grid
       Tensor<1, dim> grid_translational_velocity;
 
       // Rotational speed of rotating grid in rad/s
-      double grid_rotational_speed;
-
-      // Rotational axis of rotating grid. Similar to deal.II, we use 0=x axis,
-      // 1=y axis, 2=z axis.
-      unsigned int grid_rotational_axis;
+      Tensor<1,3> grid_rotational_speed;
```

Comment:
You introduce a rotation tensor? That's a good idea.

Reply:
Yes, it was a redundant information.